### ISSUE TITLE
not language installed bug - get_installed_lang looks like broken

### ISSUE THREAD

**Issue by pieplu** [miotto/treetagger-python#20]
Actually, if I download and install treetagger (http://www.cis.uni-muenchen.de/~schmid/tools/TreeTagger/), with a language, for exemple, french, a python call to treetagger fail, because installed language was not found.

https://github.com/miotto/treetagger-python/blob/master/treetagger.py#L135

The language file is named french.par, but treetagger-python try to find a *-utf8.par . I don't know why, maybe the naming convention was recently changed.

To temporaly fix the problem, we should copy the .par file, and add -utf8 befor the dot. Like that: `french.par -> french-utf8.par`

**Comment by miotto** [comment]
You are absolutely right, the names of the parameter files of the TreeTagger program have changed recently. I changed the source code.

Many thanks for the help.
